whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -130,20 +130,18 @@ public RdapObjectMapper(
 
     public Object map(final String requestUrl,
                       final RpslObject rpslObject,
-                      final LocalDateTime lastChangedTimestamp,
                       final Optional<AbuseContact> abuseContact) {
-        return mapCommons(getRdapObject(requestUrl, rpslObject, lastChangedTimestamp, abuseContact), requestUrl);
+        return mapCommons(getRdapObject(requestUrl, rpslObject, abuseContact), requestUrl);
     }
 
-    public Object mapSearch(final String requestUrl, final List<RpslObject> objects, final Iterable<LocalDateTime> localDateTimes, final int maxResultSize) {
+    public Object mapSearch(final String requestUrl, final List<RpslObject> objects, final int maxResultSize) {
         final SearchResult searchResult = new SearchResult();
-        final Iterator<LocalDateTime> iterator = localDateTimes.iterator();
 
         for (final RpslObject object : objects) {
             if (object.getType() == DOMAIN) {
-                searchResult.addDomainSearchResult((Domain) getRdapObject(requestUrl, object, iterator.next(), Optional.empty()));
+                searchResult.addDomainSearchResult((Domain) getRdapObject(requestUrl, object, Optional.empty()));
```

Comment:
Make sure there are tests for both these cases (i.e. result is read from the RIPE database directly, or via the Lucene index).

Reply:
I am looking at WhoisRdapServieTestIntegration and there are some tests that are already using FullTextIndex.